### DISCUSSION TITLE
How to get RBAC Token user who trigger the experiment

### DISCUSSION THREAD

**Discussion by Jim0530** [chaos-mesh/chaos-mesh#3963] [Q&A]
**Teachability, Documentation, Adoption, Migration Strategy:**

It would be helpful if i can get user who trigger the experiment!!!

**Comment by g1eny0ung** [comment]
I think we could add a custom annotation when creating an experiment in the dashboard to help someone recognize who owns the experiment (in security mode, which requires a name and a token for further use). But for the command line, this is not a very high-priority feature as there are many ways to restrict who can inject experiments.

Because Chaos Mesh doesn't have an account system. Please consider this way of distinguishing users by adding annotations as an auxiliary feature that you cannot trust too much, as Chaos Mesh does not determine the uniqueness of the identity.

**Comment by STRRL** [comment]
It's kind of auditing, maybe you could refer this: https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/
